ig-data/input/pagecontent/index.md

```diff
@@ -9,30 +9,30 @@
+<p>The ICAREdata strategy is to collect improved real-world data in a way that does not disrupt existing clinical processes and minimizes burden to the provider while enabling the collection of complete, accurate, and computable data. ICAREdata questions are answered by clinicians with the results appearing as structured phrases in the clinical note.</p>
 
-<p>ICAREdata structured phrases questions and value sets are intended to be mapped to FHIR resources compliant with the requirements specified in the mCODE and ICAREdata implementation guides. This includes mapping 1) the ICAREdata question to the appropriate FHIR resource and element, and 2) the associated value to the appropriate vocabulary defined in the specification.</p>
+<p>ICAREdata structured phrase questions and value sets are intended to be mapped to FHIR resources compliant with the requirements specified in the mCODE and ICAREdata implementation guides. This includes mapping 1) the ICAREdata question to the appropriate FHIR resource and element, and 2) the associated value to the appropriate vocabulary defined in the specification.</p>
 
 <p>The figure below illustrates how ICAREdata structured phrases relate to FHIR profiles and implementation guides.</p>
-<img src="icaredata_fhir.png" alt="ICAREdata and FHIR" width="65%" align="middle"/>
+<img src="icaredata_fhir.png" alt="ICAREdata and FHIR" width="65%" align="middle" style="margin: 0px 400px 0px 0px;"/>
 
 <p>&nbsp;</p>
 
-<p>For details on mapping guidance for specific ICAREdata structured phrases, please consult <a href=mapping_guidance.html target=_blank>ICAREdata to FHIR mapping guidance</a>.</p>
+<p>For details on mapping guidance for specific ICAREdata structured phrases, please consult <a href="mapping_guidance.html"> ICAREdata to FHIR mapping guidance</a>.</p>
 <!--
 <h1>Known Issues and Limitations</h1>
 <p>The profiles in this implementation guide may have dependencies on content in other implementation guides, namely mCODE. These dependencies may not always resolve in the profile.</p>
 -->
 
 <h1><a name="Contacts"></a>Contact Information</h1>
 
-    <p>For inquiries regarding ICAREdata Project, contact <a href="mailto:mchan@mitre.org">Miranda Chan</a>.</p>
-    <p>For questions or issues related to this implementation guide, please use our <a href="https://standardhealthrecord.atlassian.net/projects/SHRM/issues">SHR Core Models JIRA issue tracker</a>.</p>
-    <p>For questions or issues regarding the content of mCODE implementation guide, please use the <a href="https://standardhealthrecord.atlassian.net/projects/MCODE/issues/">mCODE JIRA issue tracker</a>.</p>
+<p>For inquiries regarding ICAREdata Project, contact <a href="mailto:sirivinti@mitre.org">Ammu Irivinti</a>.</p>
+<p>For questions or issues related to this implementation guide, please use our <a href="https://standardhealthrecord.atlassian.net/projects/SHRM/issues">SHR Core Models JIRA issue tracker</a>.</p>
```

Comment:
This might not be the right place for issues related to this guide. SHR Core doesn't seem appropriate. Why not (for example) use https://github.com/standardhealth/fsh-icare/issues?

Reply:
I still see standardhealthrecord.atlassian.net.